src/Transports.AspNetCore/WebSockets/WebSocketConnection.cs

```diff
@@ -276,5 +272,6 @@ protected virtual Task OnNonGracefulShutdownAsync(bool receivedCloseMessage, boo
     /// <inheritdoc/>
     public virtual void Dispose()
     {
+        GC.SuppressFinalize(this);
```

Comment:
If you want to follow dispose pattern then use
```cs
protected virtual void Dispose(bool disposing) {}
public void Dispose()
{
    Dispose(true);
    GC.SuppressFinalize(this);
}
```

Reply:
No, I don't want to.

Reply:
MS' recommended pattern: https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose

Reply:
particular https://docs.microsoft.com/en-us/dotnet/standard/garbage-collection/implementing-dispose#dispose-and-disposebool

Reply:
Nope. Just trying to ignore the compiler error.

Reply:
`#pragma warning disable`